hieradata/clients/aws.ci.jenkins.io.yaml

```diff
@@ -403,33 +403,45 @@ profile::jenkinscontroller::jcasc:
             javaHome: /opt/jdk-8
             labels:
               - ubuntu-22-amd64-maven8
-            spot: false
+              - spot
```

Comment:
We should not specify the label as it, but add a suffix `-spot` (or `-nonspot` or `-ondemand` if we follow my suggestion below).

Otherwise someone requesting a `spot` won't know if it is a Windows or Linux, or Intel, or JDK17 machine.